biz.aQute.bnd.annotation/bnd.bnd

```diff
@@ -2,5 +2,14 @@
 -include: ${workspace}/cnf/includes/jdt.bnd
 
 Bundle-Description:     bnd Annotations Library
--exportcontents:        aQute.bnd.annotation.*
 
+Export-Package: \
+	aQute.bnd.annotation.*,\
+	org.osgi.annotation.bundle,\
```

Comment:
So it's ok for `aQute.bnd.annotation` to depend on `osgi.annotation` at compile time? I wasn't sure.

Reply:
furthermore, this means that bndlib will also have an import for the package and need it at runtime to resolve in bndtools... because it embeds `aQute.bnd.annotations`.

Reply:
Of course, since the project is using the CLASS retention annotations. Not exporting them from a jar that will be going into maven central.

Reply:
Hmm, are saying the OSGi annotations appear in the type signatures (compile dependency) or the OSGI annotations are used at runtime? Bnd does not do the latter since it has its own private copies to use at runtime.

Reply:
Note the annotations, but the Enums in the package.

Reply:
it's in reference to this usage: https://github.com/bndtools/bnd/pull/3128/files#diff-25fa004735a759b33dfd907e560379adR104

Reply:
and https://github.com/bndtools/bnd/blob/master/biz.aQute.bndlib/bnd.bnd#L11